strawberryfields/gbs/plot.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Plotting and visualization
+==========================
+
+**Module name:** :mod:`strawberryfields.gbs.plot`
+
+.. currentmodule:: strawberryfields.gbs.plot
+
+This module provides functionality for visualizing GBS outputs as graphs and point processes.
+
+.. autosummary::
+    _node_coords
+    edge_coords
```

Comment:
Should we keep `edge_coords` private too?
Or we could make them both public, but given that they are very Plotly specific and I can't see users needing them - how about both private?

In which case, probably remove them from autosummary?

Reply:
Thanks for spotting this. Both should be private and removed from autosummary

Reply:
imo, I would keep it consistent (so both public, or both private). I lean towards public.

You can also have two summary sections:

```rest
Visualization functions
^^^^^^^^^^^^^^^^^^^^

The following functions produce plot.ly visualisations of graphs/subgraphs.

.. autosummary::    plot
    plot_subgraph

Auxiliary functions
^^^^^^^^^^^^^^^^

The following functions are used to generate the graph layout and data points
for the visualization functions above.

.. autosummary::
    node_coords
    edge_coords
```